src/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.cs

```diff
@@ -74,6 +74,21 @@ internal class SafeGssCredHandle : SafeHandle
     {
         private static readonly Lazy<bool> s_IsNtlmInstalled = new Lazy<bool>(InitIsNtlmInstalled);
 
+        public static SafeGssCredHandle CreateAcceptor()
+        {
+            SafeGssCredHandle retHandle = null;
+            Interop.NetSecurityNative.Status status;
+            Interop.NetSecurityNative.Status minorStatus;
+
+            status = Interop.NetSecurityNative.AcquireAcceptorCred(out minorStatus, out retHandle);
+            if (status != Interop.NetSecurityNative.Status.GSS_S_COMPLETE)
+            {
+                throw new Interop.NetSecurityNative.GssApiException(status, minorStatus, null);
```

Comment:
```suggestion
                throw new Interop.NetSecurityNative.GssApiException(status, minorStatus, helpText: null);
```

Reply:
Thanks. I copied this pattern from another place in the code. But it turns out I can simply use the 2-parameter constructor instead:
` public GssApiException(Status majorStatus, Status minorStatus)`